fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/MatcherHelper.kt

```diff
@@ -111,8 +111,10 @@ object MatcherHelper {
         for (appItem in appItems) {
             if (exists) {
                 assertTrue(appItem.waitForExists(waitingTime))
```

Comment:
You can also add an assertion error message as arguments to assertTrue/False. Ex: assertFalse("Failure error message in case it does exist", appItem.waitForExists(waitingTimeShort))

Reply:
sounds good

Reply:
Done !